test/Microsoft.AspNet.Razor.Runtime.Tests/TagHelpers/TagHelperScopeManagerTest.cs

```diff
@@ -7,7 +7,7 @@
 using Microsoft.AspNet.Razor.Runtime.TagHelpers;
 using Xunit;
 
-namespace Microsoft.AspNet.Razor.Runtime.Test.TagHelpers
+namespace Microsoft.AspNet.Razor.Runtime.Tests.TagHelpers
```

Comment:
The [guidelines](https://github.com/aspnet/Home/wiki/Engineering-guidelines#unit-test-class-naming) also say a test of the `TagHelperScopeManager` class should be in the same namespace -- `Microsoft.AspNet.Razor.Runtime.TagHelpers`. Please either leave this alone for another "Engineering Day" activity or change it to the correct namespace.

Same for the rest of the non-infrastructure test namespace changes.